### PR TITLE
LPS-191509

### PR DESCRIPTION
**Pull request by icgsbr**
# Description
> All alterations are according to WCAG.

This ticket consists on the improval on color contrast.

## Ticket Link
- [LPS-191509](https://liferay.atlassian.net/browse/LPS-191509)

## Additional informations
- Alterations were already validated with BPM design team.

### CODE REVIEW

**Comment by icgsbr** [comment]
ci:test:stable

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -167,6 +167,10 @@ body .ddm-user-view-content {
 		display: none;
 		padding: 0;
 
+		.custom-control-label-text {
+			color: #6b6c7e !important;
+		}
+
```
Comment (alinedoleron) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:173]: Hi @icgsbr, avoid using `!important` property. It worked without using it in this case.
```suggestion
		.custom-control-label-text {
			color: #6b6c7e;
		}

```

File: modules/apps/frontend-theme/frontend-theme-admin/src/css/portlet/_topper.scss [hunk 2]
```diff
@@ -1,3 +1,9 @@
 .portlet {
 	position: relative;
+
+	.taglib-workflow-status {
+		.workflow-version {
+			color: #6b6c7e !important;
+		}
+	}
```
Comment (alinedoleron) [inline modules/apps/frontend-theme/frontend-theme-admin/src/css/portlet/_topper.scss:8]: Hi @icgsbr. We can make this into our module and not change the frontend-theme module. I was able to get it working by doing the following in this file:

`/home/me/dev/projects/liferay-portal/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss`

**_general.scss:**
```
.portal-popup,
.portlet-forms-admin {

 *** existing code***
 
 .taglib-workflow-status {
		.workflow-version {
			color: #6b6c7e;
		}
	}
}
```
 Please, could you check if it works for you?
Thanks!

**Review by alinedoleron — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (icgsbr) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/css/main.scss:173]: Done!

File: modules/apps/frontend-theme/frontend-theme-admin/src/css/portlet/_topper.scss (hunk 2 above)
Comment (icgsbr) [reply modules/apps/frontend-theme/frontend-theme-admin/src/css/portlet/_topper.scss:8]: Done! Thanks for the help!

**Comment by icgsbr** [comment]
Hi @alinedoleron !

I made the requested changes and included some more classes in the contrast change scope.

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
> Hi @alinedoleron !
> 
> I made the requested changes and included some more classes in the contrast change scope.

Hi @icgsbr, good job! Just a simple adjustment, please, could you sort alphabetically the classes? Sorry for the inconvenient, I should be told you before. :grin:

Thanks!

**Comment by icgsbr** [comment]
> > Hi @alinedoleron !
> > I made the requested changes and included some more classes in the contrast change scope.
> 
> Hi @icgsbr, good job! Just a simple adjustment, please, could you sort alphabetically the classes? Sorry for the inconvenience, I should've told you before. grin
> 
> Thanks!

For sure! I'll do it right away.

**Comment by icgsbr** [comment]
ci:test:sf

**Comment by icgsbr** [comment]
ci:test:stable

**Comment by icgsbr** [comment]
ci:test:relevant

**Review by alinedoleron — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:forward:force

**Comment by icgsbr** [comment]
Additional alterations -> https://github.com/liferay-forms/liferay-portal/pull/2287

**Pull request closed without merging**